pom.xml

```diff
@@ -1176,6 +1176,11 @@
                 <artifactId>commons-lang3</artifactId>
                 <version>3.7</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>3.5</version>
```

Comment:
This dependency looks like it can be removed.  Just before this one we are already adding in `commons-lang3` version 3.7.  So, we shouldn't also need version 3.5.

Reply:
Right you are.  I suppose that the preceding dependency had not been added when I started on this patch.  I will correct this.